alpine/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine
```

Comment:
can you use a version number here, like 3.5 to avoid getting broken when alpine:4.x releases. ideally it'd be great if there was a `alpine:3` but it doesn't seem to be the case.